lib/v2/picuki/templates/video.art

```diff
@@ -0,0 +1,3 @@
+<video poster="{{ videoPoster }}" controls="controls">
```

Comment:
```suggestion
<video poster="{{ videoPoster }}" controls>
```
docs: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video#attr-controls

Reply:
Cool, Thank you.